src/domains/transaction/hooks/use-transaction-types.ts

```diff
@@ -84,14 +71,13 @@ export const useTransactionTypes = ({ wallets = [] }: TransactionTypeProperties
 				for (const wallet of wallets) {
 					allSupportedTypes.push(
 						...(wallet.transactionTypes() as string[]).filter(
-							(type) => ![MagistrateTransactionType, "ipfs", "secondSignature"].includes(type),
```

Comment:
When I was working on `ipfs` and `2nd signature` clean up removing types from this list caused issues, that is why I temporarily kept them here, if you could check with a wallet with a magistrate tx you may see the same behavior.